### PR TITLE
fix(portal-next): remove company name from footer

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-6939

## Description

Need to remove Company name from Portal footer

## Additional context

Screenshot 2024-09-24 at 15 38 55

### CODE REVIEW

File: gravitee-apim-portal-webui-next/src/components/footer/footer.component.html [hunk 1]
```diff
@@ -16,4 +16,4 @@
 
 -->
 <app-company-title [title]="title" />
-<p>© 2024 {{ company }}. All rights reserved.</p>
+<p>© 2024. All rights reserved.</p>
```
Comment (phiz71) [inline gravitee-apim-portal-webui-next/src/components/footer/footer.component.html]: I know it's not the purpose of the PR, but could it be possible to replace 2024 with the current year?
Here is how it's done in the current portal:
https://github.com/gravitee-io/gravitee-api-management/blob/5a0e988e91e1274ff52823dd863e75628d6c3fa6/gravitee-apim-portal-webui/src/app/app.component.ts#L92
https://github.com/gravitee-io/gravitee-api-management/blob/683ac06eaffd352a4364ece0b7c56e53909f4c47/gravitee-apim-portal-webui/src/app/app.component.html#L104

File: gravitee-apim-portal-webui-next/src/services/redoc.service.ts [hunk 2]
```diff
@@ -17,7 +17,6 @@ import { Injectable } from '@angular/core';
 
 import { readYaml } from '../app/helpers/yaml-parser';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (phiz71) [inline gravitee-apim-portal-webui-next/src/services/redoc.service.ts]: I think you should keep it or find the right class to replace 'any'

Comment (illichso) [reply gravitee-apim-portal-webui-next/src/services/redoc.service.ts]: Done

File: gravitee-apim-portal-webui-next/src/components/footer/footer.component.html (hunk 1 above)
Comment (illichso) [reply gravitee-apim-portal-webui-next/src/components/footer/footer.component.html]: Good point!

**Review by phiz71 — APPROVED** [review]

**Pull request merged**